src/sqlfluff/dialects/dialect_bigquery.py

```diff
@@ -383,6 +385,43 @@ class UnorderedSelectStatementSegment(ansi.UnorderedSelectStatementSegment):
     )
 
 
+class MultiStatementSegment(BaseSegment):
+    """Overriding StatementSegment to allow for additional segment parsing."""
+
+    type = "multi_statement_segment"
+    match_grammar: Matchable = OneOf(
```

Comment:
Why the OneOf()?

Reply:
Future proofing!

There's some more multi-statemnt control statements we should add here (`WHILE...DO...END WHILE`...etc) that I intend to add here, if this PR is approved. Did consider adding to this PR but then decided to get this approved first and then build upon it.